File: Lab4/LW4/LW4/Data/Models/Message.cs
namespace LW4.Data.Models;

public class Message
{
    public int Id { get; set; }
    public int ChatId { get; set; }
    public virtual Chat Chat { get; set; } = null!;

    public int UserId { get; set; }
    public virtual User User { get; set; } = null!;

    public string Text { get; set; } = null!;
    public DateTime CreatedAt { get; set; }
}

File: Lab3/MyFirstApplication/app/src/main/java/com/example/myfirstapplication/models/Message.kt
package com.example.myfirstapplication.models

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "messages")
data class Message(

    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,

    val chatId: Int,
    val userId: Int,
    val text: String,
    val createdAt: String
)